t/op/lc.t

```diff
@@ -212,12 +212,12 @@ for ("a\x{100}", "xyz\x{100}") {
 for ("A\x{100}", "XYZ\x{100}") {
     is(substr(lc($_), 0), lc($_), "[perl #38619] lc");
 }
-for ("a\x{100}", "�yz\x{100}") { # � to Ss (different length)
+for ("a\x{100}", "ßyz\x{100}") { # ß to Ss (different length)
     is(substr(ucfirst($_), 0), ucfirst($_), "[perl #38619] ucfirst");
 }
 
 #fc() didn't exist back then, but coverage is coverage.
-for ("a\x{100}", "�yz\x{100}", "xyz\x{100}", "XYZ\x{100}") { # � to Ss (different length)
+for ("a\x{100}", "ßyz\x{100}", "xyz\x{100}", "XYZ\x{100}") { # ß to Ss (different length)
```

Comment:
It looks like you've changed the encoding of this file from latin-1 to utf-8.

This makes the test not test what it's supposed to be testing - that the change in length from ucfirst of "ß" is properly handled, since perl is (as per default) reading the file as Latin-1, with the change it's reading `\xC3\x9F` instead of `\xBF`.

(github's side-by-side diff was really confusing for this.)